README.md

```diff
@@ -52,6 +52,13 @@ apt-get install gcc virtualenv python-dev libffi-dev libpq-dev
 source ./venv/bin/activate
 ```
 
+### Creating a new database migration
```

Comment:
I think this probably belongs further down - it's not part of the "Full setup" process that's being described here. Maybe next to "Updating application dependencies".